README.md

```diff
@@ -79,8 +79,11 @@ For action helpers, this will mean better currying semantics:
   + [`underscore`](#underscore)
   + [`w`](#w)
 * [Array](#array-helpers)
+  + [`array`](#array)
```

Comment:
You moved array to the top, but didn't move the section in the document itself. Either move this one back, or move the section to the correct order. I'd prefer moving this entry back to it's original place, as I don't see the need why this needs to be moved.